### ISSUE TITLE
Tts Tts.addEventListener('tts-progress', this._onProgress); is not working on android

### ISSUE THREAD

**Issue by Laxusgee** [ak1394/react-native-tts#97]
`
componentDidMount() {
    ...

    Tts.addEventListener('tts-start', this._onStart);
    Tts.addEventListener('tts-progress', this._onProgress);
    Tts.addEventListener('tts-finish', this._onFinish);
    Tts.addEventListener('tts-cancel', this._onCancel);
  }
  ...
_onProgress = (event) => {

    console.log(event)
  }
`

on ios this._onProgress works, but its not called on android

I tested on a device

**Comment by mrTuomoK** [comment]
Yeah, noticed this too. That's because the progress hasn't been implemented on Android-side at all (checked the code...)
